doc/source/examples/incompressible-flow/3d-nitsche-mixer-with-pbt-impeller/nitsche-mixer-with-pbt-impeller.rst

```diff
@@ -12,29 +9,28 @@ This example illustrates how the transient flow in a stirred-tank can be simulat
 .. seealso::
 	This example is related to this article (to be published).
```

Comment:
```suggestion
	This example is related to this article (to be published).
```
Add hyperlink to this article. It is published now.
Joachim, J., Daunais, C. A., Bibeau, V., Heltai, L., & Blais, B. (2023). A parallel and adaptative Nitsche immersed boundary method to simulate viscous mixing. Journal of Computational Physics, 488, 112189.